resources/quality/ultimaker_methodxl/um_methodxl_2a_um-pva-175_0.2mm.inst.cfg

```diff
@@ -17,60 +17,47 @@ cool_fan_enabled = False
 cool_min_speed = 5
 cool_min_temperature = =material_print_temperature
 initial_layer_line_width_factor = 300
-material_bed_temperature = 45
-material_bed_temperature_layer_0 = 50
 material_final_print_temperature = 210
 material_initial_print_temperature = 210
-material_print_temperature = 210
-material_print_temperature_layer_0 = 220
-material_standby_temperature = 160
 prime_tower_flow = 125
 raft_airgap = 0.0
-raft_base_wall_count = 4
 raft_interface_flow = 110
 raft_interface_infill_overlap = 50
 raft_interface_layers = 1
-raft_interface_line_spacing = 0.9
-raft_interface_line_width = 1
+raft_interface_line_spacing = =1.08 if raft_base_thickness == 0.5 else 0.9
+raft_interface_line_width = =1.2 if raft_base_thickness == 0.5 else 1
 raft_interface_speed = 15
-raft_interface_z_offset = -0.05
+raft_interface_z_offset = =-0.1 if raft_base_thickness == 0.5 else -0.05
 raft_surface_flow = 110
 raft_surface_infill_overlap = 50
 raft_surface_line_spacing = 0.44
-raft_surface_line_width = 0.4
 raft_surface_speed = 20
 raft_surface_thickness = 0.27
 raft_surface_z_offset = -0.075
 retraction_speed = 1.75
 speed_layer_0 = 10
 speed_prime_tower = 10
 speed_print = 50
-speed_roofing = 50
-speed_support = 50
+speed_roofing = =speed_print
+speed_support = =speed_print
 speed_support_bottom = 10
-speed_support_interface = 50
-speed_topbottom = 50
-speed_wall_0 = 50
-speed_wall_x = 50
+speed_support_interface = =speed_print
+speed_topbottom = =speed_print
+speed_wall_0 = =speed_print
+speed_wall_x = =speed_print
 support_angle = 50
 support_bottom_density = 99
-support_bottom_height = 0.406
 support_bottom_line_width = 1.2
-support_bottom_wall_count = 2
 support_conical_angle = 20
+support_conical_enabled = True
```

Comment:
Note that we still have this setting in experimental, which is usually a sign that it's not yet _fully_ trusted by us

Reply:
We released conical settings in previous Cura versions as default Method X & Method XL settings (for RapidRinse and SR30).  These values are less angled and less prone to failure.